src/datapump/jobs/geotrellis.py

```diff
@@ -278,18 +278,20 @@ def _get_result_tables(self) -> List[AnalysisResultTable]:
         bucket, prefix = get_s3_path_parts(result_path)
 
         LOGGER.debug(f"Looking for analysis results at {result_path}")
-        resp = get_s3_client().list_objects_v2(Bucket=bucket, Prefix=prefix)
-
-        LOGGER.debug(resp)
-
-        if "Contents" not in resp:
-            raise AssertionError("No results found in S3")
-
-        keys = [
-            item["Key"]
-            for item in resp["Contents"]
-            if item["Key"].endswith(".csv") and "download" not in item["Key"]
-        ]
+        paginator = get_s3_client().get_paginator("list_objects_v2")
+        pages = paginator.paginate(Bucket="bucket", Prefix="prefix")
```

Comment:
Don't we need to reference the variables and not the string literals of the `bucket` and `prefix`?
```suggestion
        pages = paginator.paginate(Bucket=bucket, Prefix=prefix)
```